docs/content/any/guides/enforcement/resource/data/data.md

```diff
@@ -0,0 +1,47 @@
+---
+authorize: authorize
+isAdmin: is_admin
+forbiddenError: ForbiddenError
+notFoundError: NotFoundError
+# TODO: change these to relative links
+authorizeLink: |-
+  [`Oso.authorize`](https://docs.osohq.com/python/reference/api/index.html#oso.Oso.authorize)
+forbiddenErrorLink: |-
+  [`ForbiddenError`](https://docs.osohq.com/python/reference/api/index.html#oso.exceptions.ForbiddenError)
+notFoundErrorLink: |-
+  [`NotFoundError`](https://docs.osohq.com/python/reference/api/index.html#oso.exceptions.NotFoundError)
+authorizationErrorLink: |-
+  an [`AuthorizationError`](https://docs.osohq.com/python/reference/api/index.html#oso.exceptions.AuthorizationError)
+exampleCall: |-
+    ```python
+    oso.authorize(user, "approve", expense)
+    ```
+getExpense: |-
+    ```python
+    def get_expense(user, expense_id):
+        expense = db.fetch(
+            "SELECT * FROM expenses WHERE id = %", expense_id)
+        oso.authorize(user, "read", expense)
```

Comment:
should this be `"approve"` to match the `authorize` call & `allow` rule above it?

```suggestion
        oso.authorize(user, "approve", expense)
```

<img width="469" alt="image" src="https://user-images.githubusercontent.com/17556281/133371582-5364e58e-a95d-4c3a-81ae-57b9a1d2516d.png">

Reply:
Ah yeah the method should probably be approveExpense, rats

Reply:
Will update everywhere